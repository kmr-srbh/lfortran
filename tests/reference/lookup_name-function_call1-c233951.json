{
    "basename": "lookup_name-function_call1-c233951",
    "cmd": "lfortran --lookup-name --no-color {infile} -o {outfile}",
    "infile": "tests/function_call1.f90",
    "infile_hash": "45acf463b1d977f1bdf3d9420205c7979fc03de58d60260cb37fd0bc",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "lookup_name-function_call1-c233951.stdout",
    "stdout_hash": "2edbce047af72876324ea358ed081ebeb4f30ff6495a73287b646a72",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}