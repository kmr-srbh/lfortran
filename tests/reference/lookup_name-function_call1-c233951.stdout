[{"kind":12,"location":{"range":{"start":{"character":5,"line":8},"end":{"character":24,"line":12}},"uri":"uri"},"name":"eval_1d","filename":"tests/function_call1.f90"}]